Add README (#27)
meilisearch/cloud-scripts: README.md

</p>

<h1 align="center">MeiliSearch Cloud Scripts</h1>

<h4 align="center">
  <a href="https://github.com/meilisearch/MeiliSearch">MeiliSearch</a> |
  <a href="https://docs.meilisearch.com">Documentation</a> |
  <a href="https://slack.meilisearch.com">Slack</a> |
  <a href="https://roadmap.meilisearch.com/tabs/1-under-consideration">Roadmap</a> |
  <a href="https://www.meilisearch.com">Website</a> |
  <a href="https://docs.meilisearch.com/faq">FAQ</a>
</h4>

<p align="center">
  <a href="https://github.com/meilisearch/cloud-scripts/blob/master/LICENSE"><img src="https://img.shields.io/badge/license-MIT-informational" alt="License"></a>
</p>

<p align="center">☁ MeiliSearch tools for the Cloud ☁</p>

**MeiliSearch Cloud Scripts** is a set of cloud-agnostic tools and scripts to improve user deployment of MeiliSearch on the cloud.

**MeiliSearch** is an open-source search engine. [Discover what MeiliSearch is!](https://github.com/meilisearch/MeiliSearch)

> 

Reviewer: ```suggestion
```

## Table of Contents <!-- omit in toc -->

- [🎁 Content of this repository](#-content-of-this-repository)
- [☁ Projects using MeiliSearch Cloud Scripts](#-projects-using-meilisearch-cloud-scripts)
- [📖 Documentation](#-documentation)
- [⚙️ Development Workflow and Contributing](#️-development-workflow-and-contributing)

## 🎁 Content of this repository

> This repository contains a few tools and scripts used mainly by the MeiliSearch team, aiming to provide our users simple ways to deploy and configure MeiliSearch in the cloud. As our heart resides on the Open Source community, we maintain several of this tools as open source repository.

Reviewer: ```suggestion
This repository contains a few tools and scripts used mainly by the MeiliSearch team, aiming to provide our users simple ways to deploy and configure MeiliSearch in the cloud. As our heart resides in the open-source community, we maintain several of these tools as open-source repositories.
```

> The goal of this repository is to provide a cloud-agnostic tools and scripts for deploying and configuring MeiliSearch.

Reviewer: ```suggestion
The goal of this repository is to provide cloud-agnostic tools and scripts for deploying and configuring MeiliSearch.
```

## ☁ Projects using MeiliSearch Cloud Scripts

> 

Reviewer: ```suggestion
```

| Cloud Provider   | Project |
|----------|:-------------|
| DigitalOcean |  [meilisearch-digitalocean](https://github.com/meilisearch/meilisearch-digitalocean/) |
| AWS |  [meilisearch-aws](https://github.com/meilisearch/meilisearch-aws/) |

> 

Reviewer: ```suggestion
```
